src/SavedSearches/Properties/CreatedByQueryString.php

```diff
@@ -2,29 +2,23 @@
 
 namespace CultuurNet\UDB3\SavedSearches\Properties;
 
-use CultuurNet\UDB3\SavedSearches\ValueObject\UserId;
-
 class CreatedByQueryString extends QueryString
 {
     /**
-     * @var UserId
+     * @param array $queryParts
      */
-    protected $userId;
-
-    /**
-     * @param UserId $userId
-     */
-    public function __construct(UserId $userId)
+    public function __construct(array $queryParts)
```

Comment:
Je kan hier ook type hinted variable arguments gebruiken:

public function __construct(string ...$queryParts)

en dan bij constructie in FixedSavedSearchRepository:
new CreatedByQueryString($this->user->mbox);
new CreatedByQueryString($this->user->mbox, $this->user->id)

Reply:
Aangepast naar splat operator.